src/Fixer/StringNotation/ExplicitStringVariableFixer.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\StringNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+final class ExplicitStringVariableFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Convert implicit variables into explicit ones in double-quoted strings.',
+            [new CodeSample('<?php $a = "My name is $name!";')],
+            implode(PHP_EOL, [
+                'The reasoning behind this rule are the following:',
+                '- When there are two valid ways of doing the same thing, using both is confusing, there should be a coding standard to follow',
+                '- PHP manual marks "$var" sintax as implicit and "${var}" syntax as explicit: explicit code should always be preferred',
+                '- Explicit syntax allows word concatenation inside strings, e.g. "${var}IsAVar", implicit doesn\'t',
+                '- Explicit syntax is easier to detect for IDE/editors and therefore has colors/hightlight with higher contrast, which is easier to read',
+            ])
```

Comment:
@keradus I don't know how you're used to create descriptions; hope you are ok with `implode(PHP_EOL, [])` syntax which allows indentation and less error-prone long strings writing.

Phrases were revisited; master of thesaurus @julienfalque please help me 🙏

Reply:
we use `\n` everywhere